src/aztec/Subsidy.sol

```diff
@@ -41,28 +41,13 @@ contract Subsidy is ISubsidy {
     error GasPerMinuteTooLow();
     error AlreadySubsidized();
     error GasUsageNotSet();
+    error NotSubsidised();
     error SubsidyTooLow();
     error EthTransferFailed();
 
     event Subsidized(address indexed bridge, uint256 indexed criteria, uint128 available, uint32 gasPerMinute);
     event BeneficiaryRegistered(address indexed beneficiary);
 
-    /**
-     * @notice Container for Subsidy related information
-     * @member available Amount of ETH remaining to be paid out
-     * @member gasUsage Amount of gas the interaction consumes (used to define max possible payout)
-     * @member minGasPerMinute Minimum amount of gas per second the subsidizer has to subsidize
-     * @member gasPerMinute Amount of gas per second the subsidizer is willing to subsidize
-     * @member lastUpdated Last time subsidy was paid out or funded (if not subsidy was yet claimed after funding)
-     */
-    struct Subsidy {
```

Comment:
Because `Subsidy struct is returned from the `getSubsidy(...)` getter and thus has to be exposed in the interface. `Beneficiary` struct is used only internally. I don't have a strong opinion about this.